host/CMakeLists.txt

```diff
@@ -109,18 +109,8 @@ add_library(oehost STATIC
 
 target_link_libraries(oehost PUBLIC oe_includes)
 
-# TODO: Replace these with `find_package` and add as dependencies to
-# the CMake package.
-if (UNIX)
-  find_library(CRYPTO_LIB NAMES crypto)
-  add_library(crypto SHARED IMPORTED)
-  set_target_properties(crypto PROPERTIES IMPORTED_LOCATION ${CRYPTO_LIB})
-
-  find_library(DL_LIB NAMES dl)
-  add_library(dl SHARED IMPORTED)
-  set_target_properties(dl PROPERTIES IMPORTED_LOCATION ${DL_LIB})
-endif ()
-
+find_package(CryptoPP REQUIRED)
+find_package(dlib REQUIRED)
```

Comment:
The line:

```
  target_link_libraries(oehost PRIVATE crypto dl Threads::Threads)
```

Must be adjusted to use the imported targets.

Reply:
After the last commit changes, the line looks like this:

`target_link_libraries(oehost PRIVATE CryptoPP:CryptoPP dlib::dlib Threads::Threads)`